frontend/src/lib/components/BranchButton.svelte

```diff
@@ -337,6 +345,7 @@
 			</ul>
 		</div>
 	{/if}
+	<LoadingIcon bind:visible={showLoadingIcon} />
```

Comment:
This might need to be outside of the div, on my machine the loading icon is in a different place than the other ones and there are scrollbars that appear when the loading icon is there.